src/libstd/sys/windows/fs.rs

```diff
@@ -635,124 +654,49 @@ pub fn copy(from: &Path, to: &Path) -> io::Result<u64> {
     Ok(size as u64)
 }
 
-#[test]
-fn directory_junctions_are_directories() {
```

Comment:
This tests was here as a regression test for https://github.com/rust-lang/rust/issues/26716, so can this stick around? It may be simple but it's exercising parts which may not be exercised elsewhere.

Either that or can you make sure this is tested in `std::fs`?